packages/invoices/src/InboundShipment/api/hooks.ts

```diff
@@ -30,9 +30,16 @@ import {
 
 export const useInboundApi = () => {
   const { storeId } = useAuthContext();
+  const keys = {
+    base: () => ['inbound'] as const,
+    detail: (id: string) => [...keys.base(), storeId, id] as const,
+    list: (params: ListParams) =>
+      [...keys.base(), storeId, 'list', params] as const,
+  };
```

Comment:
Maybe would be better to have a `stores` key which was `[ ...base(), storeId ]` to build upon.. then you could also invalidate at that level - 'all the inbounds for this store'

Also wonder if there needs to be a list v list with params.. 'invalidate all the inbound lists' .. v 'invalidate all the inbounds for this store' .. 🤷

Reply:
I'm not sure of the need for a key of 'all things in this store' though that is easier to call than invalidating a specific id and all lists. 

I think a list key would be helpful, and then extend that for the list with params version.